src/views/ManageList.jsx

```diff
@@ -93,7 +96,11 @@ export function ManageList({ data, listPath, userId, userEmail }) {
 		const response = await shareList(listPath, userId, email);
 
 		if (response) {
-			alert(shareListMessages[response.code]);
+			if (response.code === 'ok') {
```

Comment:
'ok' is a string. It is better to define an enum class.

Reply:
Hello, I added a 'Responses' object in 'ManageList.jsx' to access the value by its name, but I'm not sure if this is what you were suggesting. Do you have any suggestions to improve it or a better way to implement it?